src/main/java/seedu/address/model/note/CountryNote.java

```diff
@@ -61,6 +68,12 @@ public String toString() {
 
     @Override
     public int compareTo(CountryNote countryNote) {
-        return this.getCountry().getCountryCode().compareTo(countryNote.getCountry().getCountryCode());
+        int countryCodeComparison = getCountry().getCountryCode().compareTo(countryNote.getCountry().getCountryCode());
+        if (countryCodeComparison != 0) {
+            return countryCodeComparison;
+        } else {
+            return getNoteContent().compareTo(countryNote.getNoteContent());
```

Comment:
Might want to compare tags for the edge case where country code and note content.

Reply:
I think the tags are hard to compare (the overhead/extra code will be quite bad and messy), and its unlikely that all country code and note content will be equal but tags different, so I'll just leave it as this for now.